packages/react-charts/build-single-packages.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Is there a way to validate this JSON as it's being written?

It would be nice to not have to rely on running the `build-single-packages.js` script to know whether or not the configuration is malformed.

On top of this, it would be nice to have descriptors for each aspect of the config and documentation that shows all of the possibilities. Otherwise, we rely on reference to existing configs and a possible loss of understanding as to what we're capable of using this for without reading the `build-single-packages.js` script down the road.

Reply:
I can add this in a seperate PR.

Reply:
https://github.com/patternfly/patternfly-react/issues/8950